environment.yml

```diff
@@ -1,17 +1,18 @@
 name: cognoma-machine-learning
 dependencies:
+- conda-forge::altair=1.2.0
+- conda-forge::vega=0.4.4
 - jupyter=1.0.0
-- numexpr=2.6.0
-- numpy=1.11.1
-- pandas=0.18.1
-- python=3.5.2
-- requests=2.11.1
-- scikit-learn=0.18.0
-- scipy=0.17.1
-- seaborn=0.7.0
+- nomkl
```

Comment:
@rdvelazquez I think my error was because all my packages, besides a few (particularly `mkl`), were getting installed from the `conda-forge` channel. Specifically, scikit-learn, numpy, and scipy were coming from conda-forge and were the OpenBlas versions rather than the MKL versions. Therefore, I specified the channels for every package in https://github.com/cognoma/machine-learning/pull/103/commits/befb98dedfdc8654c5cb2f4b1e34f6f3bba66cf8.

Now frustratingly, if I `conda env remove --name=cognoma-machine-learning` and then re-install the environment, I keep getting the packages from `conda-forge`. If I rename the `environment.yml` to a different environment name, things work. So somehow my environment is persisting after removal. Frustrating, but I think we can solve later.